eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -120,7 +120,15 @@ function Confirm-ChangeLogEntry {
     else {
       $status = $changeLogEntry.ReleaseStatus.Trim().Trim("()")
       try {
-        [DateTime]$status
+        $releaseDate = [DateTime]$status
+        if ($status -ne ($releaseDate.ToString($CHANGELOG_DATE_FORMAT)))
+        {
+          LogError "Date must be in the format $($CHANGELOG_DATE_FORMAT)"
```

Comment:
Do you also need to return false in these error cases?

Reply:
Or will these end up hitting the catch block?

Reply:
You are right. I updated it.